### PR TITLE
update(ci): bumped kernel_tests to kernel-testing v0.2.0.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/kernel_tests.yaml [hunk 1]
```diff
@@ -27,9 +27,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
         with:
-          repository: alacuku/e2e-falco-tests
+          repository: alacuku/kernel-testing
```
Comment (FedeDP) [inline .github/workflows/kernel_tests.yaml:30]: TO become `falcosecurity` :D 
On hold until the repo gets donated.

**Review by Andreagit97 — APPROVED** [review]
/approve

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**